src/app/page.tsx

```diff
@@ -110,6 +110,7 @@ export default function Home() {
         <div className='pt-8'>
           <p className="p-2 my-6 text-sm border rounded-lg">Note: This app runs purely on your browser end. No images nor data will be saved by the app.</p>
           <p className='text-gray-600'>Have any feedback? <a href='https://www.x.com/sohafidz' target='_blank' className='underline cursor-pointer'>Let me know!</a></p>
+          <p className='text-gray-600'>For any bugs, please report them to our <a href='https://github.com/TechForPalestine/palestine-pfp-maker/issues' target='_blank' className='underline cursor-pointer'> GitHub repository.</a></p>
```

Comment:
Problem with this user will require GitHub account to create an issue, so not a user friendly  approach.

Reply:
It will stay as another option for those familiar with GitHub, of course, the same is the case with x too, those who are comfortable reaching out on x can do so on x, and those comfortable with GitHub can do so on GitHub.

We can also add an email option as well.